targets/unity-v2/source/Shared/Public/PluginManager.cs

```diff
@@ -92,7 +92,11 @@ private ITransportPlugin CreatePlayFabTransportPlugin()
 #endif
 #if UNITY_2017_2_OR_NEWER
             if (PlayFabSettings.RequestType == WebRequestType.UnityWww)
+#if !UNITY_2018_3_OR_NEWER // PlayFabWww will throw warnings as Unity has deprecated Www
```

Comment:
UNITY_2018_2_OR_NEWER
And.... you're really screwing with the logic by removing the body of the if statement, without removing the if statement.
The #if and the #endif need to be inside the brackets, specifically and only around the line:
```transport = new PlayFabWww();```
Because otherwise you're changing the whole logic flow of this method, likely breaking everything.